examples/GluonTS_SageMaker_SDK_Tutorial.ipynb

```diff
@@ -42,7 +42,9 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "!pip install --upgrade mxnet==1.6  git+https://github.com/awslabs/gluon-ts.git#egg=gluonts[dev]"
+    "!pip install --upgrade mxnet==1.6  git+https://github.com/awslabs/gluon-ts.git#egg=gluonts[dev]\n",
```

Comment:
This is outdated, a better one here would be 
```suggestion
    "!pip install --upgrade mxnet~=1.9  git+https://github.com/awslabs/gluon-ts.git#egg=gluonts[dev]\n",
```

Reply:
This breaks the remainder of the tutorial as the screenshot below shows: Do you know what might be the reason there? (apparently, freq does not exist anymore)

![image](https://user-images.githubusercontent.com/51889777/183445107-b34f0031-82c3-42b7-a6f5-d4b7365b7a71.png)

Reply:
Yes, just remove `freq` from the constructor of `SimpleFeedForwardEstimator` (it's not required anymore since it's not actually used by this specific model).

Reply:
Allright, thanks! The next problem we run into (I think that was also the reason we decided to stick to gluonts 1.6 back then and sf3s 0.4.2) is an issue with aiobotocore do you have an idea how to solve this? 

![image](https://user-images.githubusercontent.com/51889777/183572935-a49d9f2d-df46-48f1-9f17-159618b59559.png)

Reply:
I've never seen this issue myself, but a quick search seems to suggest that recent versions of `s3fs` got rid of this problem, see eg:
- https://stackoverflow.com/questions/68864939/s3fs-suddenly-stopped-working-in-google-colab-with-error-attributeerror-module
- https://ittutoria.net/solve-the-error-attributeerror-module-aiobotocore-has-no-attribute-aiosession/

Reply:
nevermind, just saw this

> The error persists if we upgrade s3fs to the current version

Reply:
There seems to be a problem with conflicting version dependencies which I cannot resolve immediately, need to have a closer look

![image](https://user-images.githubusercontent.com/51889777/183725563-949499b5-36d1-4769-b7ad-2be3c0ad4563.png)

![image](https://user-images.githubusercontent.com/51889777/183725598-241df4c9-bae3-4d33-be1d-44c445a99799.png)